src/ServerAdministration/WindowsService/WheelMUDService.cs

```diff
@@ -8,60 +8,37 @@
 // </summary>
 //-----------------------------------------------------------------------------
 
+using Topshelf;
```

Comment:
And I just wanted to say I was not making any comment about my volunteered time; I was just putting myself - possibly too much - in the head of somebody who may want to join this project.  No anger intended.

I actually think we should maintain a higher standard for future additions to the code base in order to stop the fly by night "I'm a developer on this" one line changes that seemed to have filled up the code base and always meant somebody got added to the credits file.  Make people earn it so it means more.

Casual additions are tracked by git-hub anyway, but a part of me wants to make the credits fileoff-limits for non-committed people.